components/ColorPickerContrastDisplay.vue

```diff
@@ -0,0 +1,119 @@
+<script lang="ts">
+export default {
+  props: {
+    colorPickerColor: {
+      type: String,
+      required: true,
+    },
+    bgColor: {
+      type: String,
+      required: true,
+    },
+    headerText: {
+      type: String,
+      required: true,
+    },
+    contrastRestriction: {
+      type: String,
+      required: true,
+    },
+  },
+  computed: {
+    contrast() {
+      return this.getContrast(this.bgColor, this.colorPickerColor);
+    },
+    isCorrect() {
+      return this.contrast.passes;
+    },
+  },
+  watch: {
+    isCorrect(newValue) {
+      this.$emit("correctContrast", newValue);
+    },
+  },
+
+  methods: {
+    setColor(event: any) {
+      this.$emit("update:color", event.target.value);
+    },
+    getContrast(hex1 = "#000000", hex2 = "#FFFFFF") {
```

Comment:
die default-werte hier dürfen weg, weil wir wollen die methode ja nie mit diesen aufrufen, oder?
```suggestion
    getContrast(hex1: string, hex2: string) {
```

Reply:
Nein, aber beim Loaden wirft er sonst Fehler